plugins/statistics.koplugin/calendarview.lua

```diff
@@ -840,6 +865,7 @@ function CalendarDayView:_populateBooks()
             value_width = value_width,
             height = self.book_item_height,
             show_parent = self,
+            use_color_rendering = self.reader_statistics.use_color_rendering,
```

Comment:
By which I mean, let the constructor (i.e., the caller) request color itself, don't try to pull the info from the other end ;).

Reply:
And cascade the attribute around subwidgets construtors if needed.